daemon/cluster/executor/container/controller.go

```diff
@@ -79,6 +79,10 @@ func (r *controller) Prepare(ctx context.Context) error {
 		return err
 	}
 
+	if err := validateMounts(r.adapter.container.task.Spec.GetContainer().Mounts); err != nil {
```

Comment:
This should probably be in the `container.go`. Validation isn't done in `Prepare`.